applications/satellite-imagery-geotiff.ipynb

```diff
@@ -18,7 +8,7 @@
     "\n",
     "This notebook shows how to perform simple calculations with a GeoTIFF dataset using XArray and Dask. We load and rescale a Landsat 8 image and compute NDVI (Normalized difference vegetation index). This can be used to distinguish green vegetation from areas of bare land or water.\n",
     "\n",
-    "We'll use an image of the Denver, USA area taken in July 2018."
+    "We'll use an image of the Denver, USA area taken in July 2018. _Note: As of May 2022, these data are no longer publicly available from S3._"
```

Comment:
Hmm that's unfortunate. Does this notebook still run then? 

I wonder if this is still publicly available, but just moved its location. There are Pangeo examples like https://gallery.pangeo.io/repos/pangeo-data/landsat-8-tutorial-gallery/landsat8.html that use Landsat 8 data. Perhaps we can update this example based on the Pangeo example notebook. Alternatively, if this notebook no longer runs we could temporarily not build/render it

Reply:
I agree, if this doesn't run, I think we should remove/hide it.

Reply:
In the interest of finishing something during our time here, I opted to remove the example. I presume it could be updated and restored by someone who knows a bit about where this data can be accessed these days.